source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -80,6 +80,53 @@ Content-Type: application/json
   ]
 }
 ```
+
+## Validating the JWT
```

Comment:
I think this whole section might be better moved further down the page underneath this section:

`## Understand your user’s core identity claim`

Which is where the original text about obtaining the key was. Just because that is the section that talks about this specific JWT and how to validate it.

Probably one to pick up when working with the tech writer though as they will be better at this than me :)

Reply:
Moved.